File: KahveQRKodOlusturucuJava-calismalarim/app/src/test/java/com/example/qrkodolusturucujava/usecasesTest/DogrulamaKoduOlusturTest.java
package com.example.qrkodolusturucujava.usecasesTest;

import com.example.qrkodolusturucujava.usecases.DogrulamaKoduOlustur;

import org.junit.Assert;
import org.junit.Before;
import org.junit.Test;

public class DogrulamaKoduOlusturTest {

    private DogrulamaKoduOlustur dogrulamaKoduOlustur;

    @Before
    public void setup() {
        dogrulamaKoduOlustur = new DogrulamaKoduOlustur();
    }

    @Test
    public void dogrulamaKoduOlusturTest() {
        String kod = dogrulamaKoduOlustur.randomIdOlustur();
        Assert.assertNotNull(kod);
    }
}

File: KahveQrOlusturucuKotlin-calismalarim/app/src/test/java/com/example/qrkodolusturucu/usecasesTEST/DogrulamaKoduOlusturTest.kt
package com.example.qrkodolusturucu.usecasesTEST

import com.example.qrkodolusturucu.usecases.DogrulamaKoduOlustur
import org.junit.Assert
import org.junit.Before
import org.junit.Test

class DogrulamaKoduOlusturTest {

    lateinit var dogrulamaKoduOlustur:DogrulamaKoduOlustur

    @Before
    fun setup(){
        dogrulamaKoduOlustur = DogrulamaKoduOlustur()
    }

    @Test
    fun dogrulamaKoduOlusturTest(){
        Assert.assertNotEquals(dogrulamaKoduOlustur.randomIdOlustur(),null)
    }

}